cogment.yaml

```diff
@@ -66,12 +66,10 @@ environment:
 
 trial:
   config_type: cogment_verse.TrialConfig
-  pre_hooks:
-    - grpc://${COGMENT_VERSE_PRETRIAL_HOOK_ENDPOINT}
 
 datalog:
```

Comment:
While we're at it we should migrate that to the prehook

Reply:
I'm not sure what you mean by this

Reply:
You can now defined the datalog in the trial params in the prehook

cf. DatalogSession.datalog_endpoint https://docs.cogment.ai/cogment/cogment-api-reference/python/#class-prehooksession

Reply:
Oh it's already done I see, we can just remove this anyways